conf/application.conf

```diff
@@ -93,4 +93,8 @@ auditing {
 }
 
 footerLinkItems = ["cookies", "privacy", "termsConditions", "govukHelp"]
-features.welsh-language-support = true
+features.welsh-language-support = false
+tracking-consent-frontend {
+  script-path = "/tracking-consent/tracking.js"
+  tracking-consent-frontend.assets-prefix = "http://localhost:12345"
```

Comment:
Should this key should be just: 
assets-prefix
not 
tracking-consent-frontend.assets-prefix